site/en/guides/pyspinel/install-wireshark.md

```diff
@@ -0,0 +1,57 @@
+# Install Wireshark
```

Comment:
This page is pretty short, why not have one page for Wireshark that covers installation and configuration?  If they are installing Wireshark for Thread purposes they have to configure it anyway.  Thoughts?